BUILD.md

```diff
@@ -52,14 +52,8 @@ prompts: <https://rustup.rs/>. This requires a download of Visual Studio (the
 selected through the VS component installer. (I'm not sure which packages are
 required; I selected them all.)
 
-Once you have rustup, you can install the Rust toolchain needed to compile
-Artichoke.
-
-```sh
-rustup toolchain install "$(cat rust-toolchain)"
-rustup component add rustfmt
-rustup component add clippy
-```
```

Comment:
Can we rework this section to have folks manually install `rustfmt` and `clippy`? I also think the rustup command can be condensed to this:

```console
$ rustup component add rustfmt clippy
info: component 'rustfmt' for target 'x86_64-apple-darwin' is up to date
info: component 'clippy' for target 'x86_64-apple-darwin' is up to date
```